### ISSUE TITLE
Dealing with comments and suggestions in the inline editor 

### ISSUE THREAD

**Issue by daniel-montalvo** [microsoft/vscode-pull-request-github#1683]
Thanks for the very good work you are doing with this extension!

There seems not to be an apparent way to deal with comments in the inline editor for screen reader users (or I am totally missing it). There is a tree view for changes but not so for comments.

1. Is there an implemented workflow for inserting, replying, and suggesting using the VSCode inline editor that takes into account screen reading technologies? If so, please point me to some guidance on how to accomplish it.
2. If not, I would really appreciate you could work on this, as adding, managing, replying, and suggesting could be much easier from an inline local editor than it actually is from within the GitHub web interface for reviews.
